src/clients/SpokePoolClient.ts

```diff
@@ -439,41 +441,14 @@ export class SpokePoolClient extends BaseAbstractClient {
     low: number;
     high: number;
   }> {
-    if (initLow > initHigh) {
```

Comment:
Is this function still an async one?

Reply:
It is currently not needed, but I do think we should do this in a separate PR.